ios/brave-ios/Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -872,6 +879,23 @@ class Tab: NSObject {
     webView.customUserAgent = desktopMode ? UserAgent.desktop : UserAgent.mobile
   }
 
+  func addTabHelper(_ helper: TabHelper) {
+    let helperName = Swift.type(of: helper).tabHelperName
+    if tabHelpers[helperName] != nil {
+      assertionFailure("Tab Helper: \(helperName) already attached to this tab.")
+    }
+
+    self.tabHelpers[helperName] = helper
+  }
+
+  func getTabHelper(named name: String) -> TabHelper? {
+    self.tabHelpers[name]
+  }
+
+  func removeTabHelper(named name: String) {
+    self.tabHelpers.removeValue(forKey: name)
+  }
+
```

Comment:
Lets just dump all of this `TabHelper` stuff and just use a standard property on `Tab` since we're not even matching Chromium's definition of a tab helper at this point anyways, we can always move towards that direction later

Reply:
All of it removed. Added directly to `Tab`.